_docs/_latest/develop/branch-management.md

```diff
@@ -44,19 +50,29 @@ After the EAP is complete, a Release Candidate (RC) will be created, like
 the team will merge the completed code into `N.N-dev`, then into
 `N.N-master`, then into `dev`, and finally `master`.
 
-Following is an example of this workflow for version 2.0:
+Following is an example of this workflow for future versions:
+
+* `3.0-dev`: A daily development branch.
+* `3.0-eap1`: A branch used for Early Access testing.
+* `3.0-rc1`: A possible release candidate.
+* `3.0-rc2`: A second release candidate with additional changes.
+* `master`: The final released version of v2.0.
 
-* `2.0-dev`: A daily development branch.
-* `2.0-eap1`: A branch used for Early Access testing.
-* `2.0-rc1`: A possible release candidate.
-* `2.0-rc2`: A second release candidate with additional changes.
-* `2.0-master`: The final released version of v2.0.
+Before launching a new version, the existing `dev` and `master` will be moved
+and renamed as `(N-1).0-dev` and `(N-1).0-master`.  Existing `N.N-dev`
+will be moved and renamed to `dev`. A new `master` branch will be created from
+`dev` branch.  This is so that the `dev` and `master` branches are always
+maintained as the canonical branches, and all the commit histories and tags
+are retained.
 
-After the support period has passed for the previous version, the
-next-generation `dev` and `master` branches will be merged into the
-`dev` and `master` branches. Then, the `N.N-<name>` branches will be deleted. 
-This is so that the `dev` and `master` are always maintained as the canonical
-branches, and all the commit histories are retained.
+For example, when version 3.0 is ready to be launched:
+
+```
+dev --> 2.0-dev
```

Comment:
This isn't self explanatory, provide commentary with the a `# Comment.` before each line.

Reply:
how about this?